ibmcloud/README.md

```diff
@@ -359,6 +359,44 @@ $ terraform apply
 >    $ ibmcloud is images --visibility=private
 >    ```
 
+### Enabling Attestation agent and Authenticated Registry
+> **Prerequisites:**
+> - An ibmcloud worker node using the cloud-api-adaptor
+> - A `auth.json` file with your credentials in, based off: (containers-auth.json.5.md)[https://github.com/containers/image/blob/main/docs/containers-auth.json.5.md]
+> - SSH'd into the worker node: `ssh root@floating-ip-of-worker-node`
+
+**Setting up the attestation agent:**
+- `cd ~/cloud-api-adaptor/podvm/files/etc`
+- Base64 your `auth.json`, this can be done by doing `cat auth.json | base64 -w 0`
```

Comment:
Hi, it seems podvm/copy-files.sh doesn't copy this file, does it? maybe i missed something..